src/_http2.js

```diff
@@ -0,0 +1,39 @@
+var http2 = require('http2')
+
+function request({
+  url,
+  body,
+  headers,
+  timeout,
+  onError,
+  scheme = 'https',
+} = {}) {
+  var method = http2.constants.HTTP2_METHOD_POST
+  var client = http2.connect(url)
+
+  client.on('error', onError)
+
+  var request = client.request(
+    {
+      ...headers,
```

Comment:
Is this ES5 compatible?

Reply:
Good catch, I think this is not.

Reply:
atm indeed "Uncaught ReferenceError: http2 is not defined"